bin/test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+
+
+if [[ (! -z "${AWS_ACCESS_KEY_ID}"  && ! -z "${AWS_ACCESS_KEY_ID}" )|| ( -f ~/.aws/config && -f ~/.aws/credentials ) ]]; then
+    echo did a thing
+else
+    echo didnt
+fi
```

Comment:
What is the use case for this file? If it's meant to be included can you include a final newline as well?

Reply:
oh, nope, totally just random testing that got pulled in.

Reply:
Yep random testing.

Reply:
Removed now